packages/gitgraph-core/src/user-api/gitgraph-user-api.ts

```diff
@@ -139,6 +158,7 @@ class GitgraphUserApi<TNode> {
     }
 
     this._graph.tags.set(name, commitHash);
+    this._graph.tagStyles[name] = style;
```

Comment:
Simplest solution I found was to store the style in a map, then to retrieve it when we compute commit style (tag uses commit style as fallback).